gdbstub.c

```diff
@@ -2248,8 +2248,13 @@ static void handle_query_xfer_auxv(GArray *params, void *user_ctx)
     offset = get_param(params, 0)->val_ul;
     len = get_param(params, 1)->val_ul;
     ts = gdbserver_state.c_cpu->opaque;
+#if defined(CONFIG_BSD_USER)
```

Comment:
I need to check on this because I was sure we saved this... that's the real fix here, imho, if we don't already.